libnetwork/cni/network.go

```diff
@@ -295,6 +295,12 @@ func (n *cniNetwork) DefaultInterfaceName() string {
 	return cniDeviceName
 }
 
+// NetworkInfo return the network information about binary path,
+// package version and program version.
+func (n *cniNetwork) NetworkInfo() map[string]interface{} {
+	return nil
```

Comment:
we should fill in basic info about cni, at least we can query `containernetworking-plugins` package.